index.html

```diff
@@ -12,50 +12,188 @@
+        <div class="card-img"></div>
+        <div class="box">
+          <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+          <ul class="tags">
+            <li class="tag-item">Ruby on rails</li>
+            <li class="tag-item">CSS</li>
+            <li class="tag-item">JavaScript</li>
+            <li class="tag-item">HTML</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+
+      <div class="grid-card">
+        <div class="card-img"></div>
+        <div class="box">
+          <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+          <ul class="tags">
+          <li class="tag-item">Ruby on rails</li>
+          <li class="tag-item">CSS</li>
+          <li class="tag-item">JavaScript</li>
+          <li class="tag-item">HTML</li>
+        </ul>
+        <button>See Project</button>
+        </div> 
+      </div>
+
+      <div class="grid-card">
+        <div class="card-img"></div>
+        <div class="box">
+          <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+          <ul class="tags">
+            <li class="tag-item">Ruby on rails</li>
+            <li class="tag-item">CSS</li>
+            <li class="tag-item">JavaScript</li>
+            <li class="tag-item">HTML</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+
+      <div class="grid-card">
+        <div class="card-img"></div>
+        <div class="box">
+          <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+          <ul class="tags">
+            <li class="tag-item">Ruby on rails</li>
+            <li class="tag-item">CSS</li>
+            <li class="tag-item">JavaScript</li>
+            <li class="tag-item">HTML</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+
+      <div class="grid-card">
+        <div class="card-img"></div>
+        <div class="box">
+          <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+          <ul class="tags">
+            <li class="tag-item">Ruby on rails</li>
+            <li class="tag-item">CSS</li>
+            <li class="tag-item">JavaScript</li>
+            <li class="tag-item">HTML</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <!-- About myself -->
+  <section class="about-me">
+    <div class="title-card">
+      <h2 class="about-me-title">About Me</h2>
+      <p class="about-me-subtitle"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don't hestiate to contact me.</p>
+        <button class="about-me-btn">Get My Resume</button>
+    </div>
+    <div class="grid-container">
+      <div class="lang-card about-box">
+        <img src="images/Icons.png" alt="icon languages">
+        <h4 class="lang-title">Languages</h4>
```

Comment:
- [ ] Please implement the background images as shown in the design specification 👇 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/185723901-c2efbe9e-03e8-41a3-86eb-079ca7564c60.png)

### Figma Design
![image](https://user-images.githubusercontent.com/46839548/185723877-37d1dbbb-a534-42db-8804-985d39c74352.png)